script/status.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs')
+const path = require('path')
+const request = require('request')
+
+const projectKey
```

Comment:
Use the `dotenv-safe` to get the key

Reply:
Yeah I agree. Here's how to set up `dotenv-safe`

```sh
npm i -D dotenv-safe
echo "CROWDIN_KEY=" > .env.example
echo "CROWDIN_KEY=s3cr3tz" > .env
echo ".env" >> .gitignore
```

Then in status.js:

```js
require('dotenv-safe').load()
const key = process.env.CROWDIN_KEY
```

Reply:
done :zap: